.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,92 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[State]: Title of the bug"
```

Comment:
What is "[State]"?

Reply:
Some string helping to categorize the issue. 
We can remove it as we have labels :)